gae/room/ndb_models.py

```diff
@@ -1203,10 +1203,14 @@ def name(self):
     return self.position_name
 
   def GetHourlyRate(self, activity_date):
-    return _GetRateFromArray(self.hourly_rate, self.hourly_rate_after_date, activity_date)
+    if self.hourly_rate_after_date:
+      return _GetRateFromArray(self.hourly_rate, self.hourly_rate_after_date, activity_date)
+    return 0.0
```

Comment:
This makes testSiteBudget (test_ndb_models.ModelsTest) _FAIL_.  
Is there another way to keep the staff positions up to date without affecting other models?